pandas/core/common.py

```diff
@@ -1888,60 +1890,58 @@ def _possibly_convert_objects(values, convert_dates=True,
                               convert_timedeltas=True):
     """ if we have an object dtype, try to coerce dates and/or numbers """
 
+    # If 1 flag is coerce, ensure 2 others are False
+    conversions = (convert_dates, convert_numeric, convert_timedeltas)
+    if 'coerce' in conversions:
+        coerce_count = sum([c == 'coerce' for c in conversions])
+        if coerce_count > 1:
+            raise ValueError("'coerce' can be used at most once.")
+
+        false_count = sum([not c for c in conversions])
```

Comment:
I don't think this is worth the effort to show the warning. I would take it out (of course the ValueError above is the most important item).